app/models/custom_button.rb

```diff
@@ -143,4 +146,19 @@ def self.get_user(user)
     raise _("Unable to find user '%{user}'") % {:user => user} if user.nil?
     user
   end
+
+  def clone(options = {})
+    options[:guid] ||= MiqUUID.new_guid
```

Comment:
@bdunne Is there a use case where we do not want to change the guid?  The model validates uniqueness and presence of `:guid`.

Should this just be: `options[:guid] = MiqUUID.new_guid`